src/modules/compliance/src/lib/procedures/EnsureRootPath.cpp

```diff
@@ -0,0 +1,82 @@
+#include <CommonUtils.h>
+#include <Evaluator.h>
+#include <grp.h>
+#include <iostream>
+#include <pwd.h>
+#include <sstream>
+#include <string>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <vector>
+
+namespace compliance
+{
+AUDIT_FN(EnsureRootPath)
+{
+    UNUSED(args);
+    UNUSED(context);
+
+    const int maxPerm = 0777 & ~0022;
+    auto rootEnv = context.ExecuteCommand("sudo -Hiu root env");
```

Comment:
what if `sudo` isn't present or we are already `root`?

Reply:
We not only need to be uid0, we also need to load all the environment etc etc. that the root loads by default. This is how CIS does it, so I wouldn't reinvent the wheel here.